benchmarks/perf-tool/okpt/test/steps/steps.py

```diff
@@ -325,11 +320,101 @@ def action(doc_id):
     def _get_measures(self) -> List[str]:
         return ['took']
 
+    @abstractmethod
+    def handle_data_bulk(self, action, i):
+        pass
+
 
-class QueryStep(OpenSearchStep):
+class IngestStep(BaseIngestStep):
     """See base class."""
 
-    label = 'query'
+    label = 'ingest'
+
+    def handle_data_bulk(self, action, i):
+        partition = self.dataset.read(self.bulk_size)
+        if partition is None:
+            return
+        body = bulk_transform(partition, self.field_name, action, i)
+        bulk_index(self.opensearch, self.index_name, body)
+
+
+class IngestStepExtended(BaseIngestStep):
```

Comment:
I think a better name would be IngestMultiFieldStep. I dont think extended is intuitive

Reply:
agree, Extended isn't very. intuitive but I couldn't figure out better name. IngestMultiFieldStep sounds reasonable